app/src/main/java/com/nilhcem/blenamebadge/ui/custom/PreviewBadge.kt

```diff
@@ -85,14 +88,14 @@ class PreviewBadge : View {
     override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
         super.onMeasure(widthMeasureSpec, heightMeasureSpec)
         val ratioHeight = 1
-        val ratioWidth = 3.42
```

Comment:
Was this required?

Reply:
Yes, This is a must. 3.42 will have issues in larger xxxhdpi devices as the width is way too much. I can confirm this issue, as I own a xxxhdpi device

Reply:
Ok. I think the rest looks fine.

Reply:
there will be a 14% decrease in height to be specific.